providers/implementations/rands/drbg.c

```diff
@@ -235,7 +235,7 @@ static size_t get_entropy(PROV_DRBG *drbg, unsigned char **pout, int entropy,
      *       a warning in some static code analyzers, but it's
      *       intentional and correct here.
      */
-    bytes = drbg->parent_get_seed(drbg->parent, pout, drbg->strength,
+    bytes = drbg->parent_get_seed(drbg->parent, pout, entropy,
```

Comment:
`entropy > 0 ? entropy : drbg->strength` might be a safer option here.

Reply:
`entropy > drbg->strength ? entropy : drbg->strength` ? if you want to ensure previous behavior remains in place.

Reply:
Ack. @paulidale The next two argument are min_len and max_len => what do those do? do they not protect floor and ceiling generated entropy, or not?

Reply:
I don't remember exactly.  I think these come via NIST's definitions.